<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VP1EventDisplayEditorForm</class>
 <widget class="QWidget" name="VP1EventDisplayEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>923</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event Display Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_left">
     <item>
      <widget class="QGroupBox" name="groupBox_getTab">
       <property name="title">
        <string>Get image from Tab</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QComboBox" name="comboBox_tabNumber"/>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_imageWidth">
          <property name="toolTip">
           <string>Specify the width of the image generated from the selected tab</string>
          </property>
          <property name="suffix">
           <string> px</string>
          </property>
          <property name="maximum">
           <number>4000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_get">
          <property name="text">
           <string>Get!</string>
          </property>
          <property name="icon">
           <iconset resource="vp1.qrc">
            <normaloff>:/vp1/icons/icons/redo_32x32.png</normaloff>:/vp1/icons/icons/redo_32x32.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_bkg">
       <property name="title">
        <string>Background Color</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioButton_black">
          <property name="text">
           <string>Black</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_white">
          <property name="text">
           <string>White</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_transparent">
          <property name="text">
           <string>Transparent</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_logo">
       <property name="title">
        <string>ATLAS Logo</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>style</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QRadioButton" name="radioButton_gray">
                   <property name="text">
                    <string>Gray</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButton_blue">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Blue</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QGroupBox" name="groupBox_4">
                <property name="title">
                 <string>width</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QRadioButton" name="radioButton_300px">
                   <property name="text">
                    <string>300 px</string>
                   </property>
                   <property name="checked">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButton_450px">
                   <property name="text">
                    <string>450 px</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButton_600px">
                   <property name="text">
                    <string>600 px</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButton_800px">
                   <property name="text">
                    <string>800 px</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButton_1000px">
                   <property name="text">
                    <string>1000 px</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButton_1200px">
                   <property name="text">
                    <string>1200 px</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_addLogo">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Add ATLAS logo</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_labels">
       <property name="title">
        <string>Labels</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Add a text label</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>GroupBox</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_17">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QRadioButton" name="radioButton_customText">
                    <property name="text">
                     <string>custom text</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_eventDetails">
                    <property name="toolTip">
                     <string>Add a label to the picture, with run number, event number and event timestamp</string>
                    </property>
                    <property name="text">
                     <string>event details</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Add label</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_customLabelText"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QRadioButton" name="radioButton_blackText">
                 <property name="text">
                  <string>Black</string>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_whiteText">
                 <property name="text">
                  <string>White</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_addText">
                 <property name="text">
                  <string>Add label</string>
                 </property>
                 <property name="icon">
                  <iconset resource="vp1.qrc">
                   <normaloff>:/vp1/icons/icons/list_add_font.png</normaloff>:/vp1/icons/icons/list_add_font.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QGroupBox" name="groupBox_font">
                 <property name="title">
                  <string>font</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <widget class="QRadioButton" name="radioButton_monaco">
                    <property name="text">
                     <string>Courier</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_helvetica">
                    <property name="text">
                     <string>Helvetica</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_times">
                    <property name="text">
                     <string>Times</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_size">
                 <property name="title">
                  <string>size</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_10">
                  <item>
                   <widget class="QRadioButton" name="radioButton_size12">
                    <property name="text">
                     <string>12</string>
                    </property>
                    <property name="checked">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_size24">
                    <property name="text">
                     <string>24</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_size32">
                    <property name="text">
                     <string>32</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_size44">
                    <property name="text">
                     <string>44</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_sizeCustom">
                    <property name="text">
                     <string>custom</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_size"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_style">
                 <property name="title">
                  <string>style</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_14">
                  <item>
                   <widget class="QRadioButton" name="radioButton_light">
                    <property name="text">
                     <string>Light</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_normal">
                    <property name="text">
                     <string>Normal</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_bold">
                    <property name="text">
                     <string>Bold</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_italics">
                    <property name="text">
                     <string>Italic</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_black_2">
                    <property name="text">
                     <string>Black</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QGroupBox" name="groupBox_objectList">
       <property name="title">
        <string>Graphics object lists</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_removeObj">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Remove object</string>
          </property>
          <property name="icon">
           <iconset resource="vp1.qrc">
            <normaloff>:/vp1/icons/icons/button_cancel_32x32.png</normaloff>:/vp1/icons/icons/button_cancel_32x32.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="framePreview">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_actions">
       <property name="title">
        <string>Actions</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="pushButton_print">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Print Picture</string>
          </property>
          <property name="icon">
           <iconset resource="vp1.qrc">
            <normaloff>:/vp1/icons/icons/fileprint_32x32.png</normaloff>:/vp1/icons/icons/fileprint_32x32.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_save">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Save Picture</string>
          </property>
          <property name="icon">
           <iconset resource="vp1.qrc">
            <normaloff>:/vp1/icons/icons/filesave_32x32.png</normaloff>:/vp1/icons/icons/filesave_32x32.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widgetView" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="vp1.qrc"/>
 </resources>
 <connections/>
</ui>
